### DISCUSSION TITLE
10.20.3 سرور ها داخل سابسکریشن پی اچ پی اضافه نمیشه

### DISCUSSION THREAD

**Discussion by mehrdadrm** [hiddify/Hiddify-Manager#3980] [Bug باگ: please use issue]
سلام وقت بخیر من از نسخه 10.14 اپدیت کردم به این مشکل خوردم. حذف نصب هم کردم جواب نداد 
![php](https://github.com/hiddify/Hiddify-Manager/assets/166922590/00016c95-e15e-4f56-a9c5-73e11edaced9)

**Comment by mokorocko** [comment]
same here
